public/scripts/project.js

```diff
@@ -1,149 +1,155 @@
 'use strict';
 
-function Project(name, description, url, backgroundimage) {
-  this.name = name;
-  this.description = description;
-  this.url = url;
-  this.backgroundimage = backgroundimage;
-}
-
-Project.all = [];
-
-Project.prototype.toHtml = function() {
-  let template = Handlebars.compile($('#project-template').text());
-  return template(this);
-};
-
-Project.load = function(projectData){
-  projectData.forEach(function(proj) {
-    Project.all.push(new Project(proj.name, proj.description, proj.url, proj.backgroundimage));
-  })
-}
-
-Project.fetch = function() {
-  if(localStorage.projectData){
-    Project.load(JSON.parse(localStorage.projectData));
-    projectView.initIndexPage();
-  } else {
-    $.ajax({
-
-      url: './data/project_file.json',
-      dataType: 'json',
-      function (err) {
-        console.error(err)
-      },
-    }).done(function(data) {
-      localStorage.setItem('projectData', JSON.stringify(data));
-      Project.load(JSON.parse(localStorage.projectData));
-      projectView.initIndexPage();
-    })
+((module) => {
+
+  function Project(name, description, url, backgroundimage) {
```

Comment:
if you actually pass a placeholder here like 'opts', you can assign them like:
function Project(opts){
this.name = opts.name
...
}
Then when you are actually make project objects. You would only need to pass the data.
Line 21?: Project.all = data.map(proj => {
 +      return new Project(proj);
 +    });

This in theory should parse all those values.